### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by vivekmore**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,17 +1,18 @@
-safe:        true
+safe: true
 markdown: kramdown
 exclude: [node_modules, bower_components, vendor]
 url: https://www.jhipster.tech
 github_repository_url: https://github.com/jhipster/jhipster.github.io
 kramdown:
   input: GFM
-gems:
+plugins:
```
Comment (ruddell) [inline _config.yml:8]: This change has caused issues in the past, not sure how to reproduce it though.  See https://github.com/jhipster/jhipster.github.io/pull/557 and https://github.com/jhipster/jhipster.github.io/pull/776

Comment (vivekmore) [reply _config.yml:8]: Understood, I'll revert this.
Should I add a note/comment so another such attempt is avoided?

Comment (vivekmore) [reply _config.yml]: done and done! ✔️

**Comment by vivekmore** [comment]
@ruddell - is there anything else you'd like me to address?

File: _config.yml (hunk 1 above)
Comment (ruddell) [reply _config.yml:8]: I don't the change or comment.  I still see `gems` being changed to `plugins`, am I missing the change?

Other spacing/renamings look OK to me, did not test locally.

Comment (vivekmore) [reply _config.yml:8]: I had reverted it, but it looks like I goofed up in one of the rebases since ... (or I never reverted it - which is also likely 😨)

Comment (vivekmore) [reply _config.yml]: I've reverted it now

**Comment by pascalgrimaud** [comment]
Thanks a lot @vivekmore

**Pull request merged**